AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -254,6 +254,15 @@ - (void)setFrame:(CGRect)rect
       CGPoint newPosition = CGPointZero;
       ASBoundsAndPositionForFrame(rect, origin, anchorPoint, &newBounds, &newPosition);
 
+      if (ASIsCGRectValidForLayout(newBounds) == NO) {
+        ASDisplayNodeAssert(NO, @"-[ASDisplayNode setFrame:] - The new bounds are invalid and unsafe to be set.");
```

Comment:
Actually, do this: consolidate to 1 if (bounds valid == NO || position valid == NO), and use just one assertion.  Since the user is providing a value to setFrame:, we can use the log with StringFromCGRect(rect), rather than breaking it into bounds / position for the log itself.